### DISCUSSION TITLE
How can I make VS display the *verified* files in Solution Explorer

### DISCUSSION THREAD

**Discussion by ModernRonin** [VerifyTests/Verify#672] [Q&A] [answered]
I've tried adding them explicitly to the project, weirdly that does nothing - the csproj is not touched.

So I guess there's some project-file trickery I could use, but maybe someone has already solved this?

**Comment by SimonCropp** [comment, marked as answer]
verified file are nested under their parent file
![image](https://user-images.githubusercontent.com/122666/195959897-39fe1bce-62d7-4ed3-a8ee-d78b51ce40b6.png)

**Reply by ModernRonin** [reply]
oh my, I'm so blind! thanks a lot!

**Reply by SimonCropp** [reply]
no worries. it isn’t exactly obvious.
